data/publiccloud/terraform/sap/azure.tfvars

```diff
@@ -319,7 +318,7 @@ drbd_image_uri = "https://eisleqasapopenqa.blob.core.windows.net/sle-images/%SLE
 drbd_enabled = false
 
 # Each drbd cluster host IP address (sequential order). If it's not set the addresses will be auto generated from the provided vnet address range
-drbd_ips = ["10.74.1.21", "10.74.1.22"]
+drbd_ips         = ["10.74.1.21", "10.74.1.22"]
```

Comment:
Is this tab needed ?

Reply:
I wondered as well, but all other grouped vars in the file (most of which are commented out, but there are a couple uncommented too) use tabs to look uniform, so I think it's fine for consistency

Reply:
It is result of `terraform fmt`. Sorry was not intentional (I have it configured in my editor) but I hope at the same time it is not so annoying.